pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -227,21 +239,13 @@ func (rc *reconciler) reconcile() {
 			}
 
 			// Trigger detach volume which requires verifing safe to detach step
-			// If timeout is true, skip verifySafeToDetach check
+			// If isSafeToDetach is true, skip verifySafeToDetach check
 			glog.V(5).Infof(attachedVolume.GenerateMsgDetailed("Starting attacherDetacher.DetachVolume", ""))
-			verifySafeToDetach := !timeout
-			if utilfeature.DefaultFeatureGate.Enabled(features.NodeShutdown) {
-				isShutdownNode := rc.desiredStateOfWorld.NodeIsShutdown(attachedVolume.NodeName)
-
-				if isShutdownNode {
-					glog.V(1).Infof("node %v is shutdown, the volume is safe to detach.", attachedVolume.NodeName)
-					verifySafeToDetach = !isShutdownNode
-				}
-
-			}
+			verifySafeToDetach := !isSafeToDetach
+			glog.V(1).Infof("node %v is shutdown, the volume is safe to detach.", attachedVolume.NodeName)
```

Comment:
node is not always shutdown if we go to this row. Little bit misleading logprint, maybe we could put this to some another place